### ISSUE TITLE
equal-graphs? does not strictly compare attributes

### ISSUE THREAD

**Issue by chairmank** [Engelberg/ubergraph#4]
This is a question and possibly an enhancement request. The implementation of `equiv` for `Ubergraph` seems to treat two graphs that have the same graph structure as equal, even when the attributes on their nodes and/or edges are not all equal. Is this intentional?

I would find it more intuitive if

```
(let [g (uber/graph :a)]
  (= (uber/set-attrs g :a {:counter 1})
     (uber/set-attrs g :a {:counter 2})))
```

returned `false` instead of `true`.

**Comment by Engelberg** [comment]
Your intuition makes sense.  I am unavailable to work on it this week,  but
next week I will investigate the issue and either make the change you
suggest or provide a justification for the current behavior.  Thanks for
pointing this out.

**Comment by chairmank** [comment]
Thanks. I can contribute a patch if you decide that the current behavior isn't correct.

**Comment by Engelberg** [comment]
Fixed in 0.1.5.

Thanks for the report.

**Comment by Engelberg** [comment]
P.S. edge attributes already worked as expected, it was only node attributes that still needed to be dealt with.
